src/main/java/ch/njol/skript/expressions/ExprInventoryInfo.java

```diff
@@ -52,8 +52,9 @@ public class ExprInventoryInfo extends SimpleExpression<Object> {
 	
 	static {
 		Skript.registerExpression(ExprInventoryInfo.class, Object.class, ExpressionType.PROPERTY,
-				"(" + HOLDER + "¦holder[s]|" + VIEWERS + "¦viewers|" + ROWS + "¦[amount of] rows|" + SLOTS + "¦[amount of] slots)" + " of %inventories%",
-				"%inventories%'[s] (" + HOLDER + "¦holder[s]|" + VIEWERS + "¦viewers|" + ROWS + "¦[amount of] rows|" + SLOTS + "¦[amount of] slots)");
+				"(" + HOLDER + "¦holder[s]|" + VIEWERS + "¦viewers|" + ROWS + "¦[amount of] rows|" + SLOTS + "¦([amount of] slots|inventory size))" + " of %inventories%",
+				"%inventories%'[s] (" + HOLDER + "¦holder[s]|" + VIEWERS + "¦viewers|" + ROWS + "¦[amount of] rows|" + SLOTS + "¦([amount of] slots|inventory size))"
```

Comment:
I think `inventory size` is quite ambiguous and might return what somebody doesn't expect. Having the rows/slots part explicit was probably better.